.gitlab-ci.yml

```diff
@@ -0,0 +1,74 @@
+stages:
+  - build
+  - test
+  - report
+
+before_script:
+  - sh -c 'echo "deb [arch=amd64,arm64] http://repo.ros2.org/ubuntu/main `lsb_release -cs` main" > /etc/apt/sources.list.d/ros2-latest.list'
+  - curl http://repo.ros2.org/repos.key | apt-key add -
+  - apt-get update
+  - apt-get install -y python3-colcon-common-extensions
+  - source /opt/ros/crystal/setup.bash
+
+build_launchtest:
+  stage: build
+  image: osrf/ros2:nightly
+  script:
+    - colcon build --packages-up-to apex_launchtest apex_launchtest_cmake
+  artifacts:
+    paths:
+      - .
+
+build_all:
+  stage: build
+  image: osrf/ros2:nightly
+  script:
+    - colcon build
+  artifacts:
+    paths:
+      - .
```

Comment:
Nitpick: will it be better to upload only the `build` and `install` folders? If you upload everything under `.`, `.git` will also be uploaded and then override the same folder in job `test_all`. Seems a potential bug to me.

Reply:
Yes, I can do that